copier/cli.py

```diff
@@ -189,6 +191,17 @@ def data_switch(self, values: List[str]) -> None:
             values: The list of values to apply.
                 Each value in the list is of the following form: `NAME=VALUE`.
         """
+        files = [e for e in values if "=" not in e]
+        values = [e for e in values if "=" in e]
```

Comment:
Please avoid using meaningles var names such as `e`, for the sake of readability.